Sources/PointFree/Account/Account.swift

```diff
@@ -51,10 +61,16 @@ private func fetchAccountData<I, A>(
 
 let accountView = View<(Stripe.Subscription?, [Database.TeamInvite], [Database.User], [Database.EmailSetting], Database.User)> { subscription, teamInvites, teammates, emailSettings, currentUser in
 
-  titleRowView.view(unit)
-    <> profileRowView.view((currentUser, emailSettings))
-    <> subscriptionRowView.view((subscription, teamInvites, teammates))
-    <> logoutView.view(unit)
+  gridRow([
+    gridColumn(sizes: [.mobile: 12, .desktop: 8], [style(margin(leftRight: .auto))], [
+      div([`class`([Class.padding([.mobile: [.all: 3], .desktop: [.all: 4]])])],
```

Comment:
I copied and pasted this type of row (mobile 12, desktop 8, auto margins, lots of padding) a few times just to get tests passing, but it could also make sense to either create a new type of layout for columnar pages (e.g. account, terms, etc..), or create an option in `SimplePageLayoutData`